src/main/java/com/jagrosh/jmusicbot/settings/SettingsManager.java

```diff
@@ -31,12 +32,14 @@
  */
 public class SettingsManager implements GuildSettingsManager
 {
-    private final static double SKIP_RATIO = .55;
     private final HashMap<Long,Settings> settings;
+    private final BotConfig config;
 
-    public SettingsManager()
+    public SettingsManager(BotConfig config)
```

Comment:
Any references to the `BotConfig` in this class should become unnecessary, once my suggestions are applied. Please remove them.